app/tests/unit/components/Form/ProjectMilestoneReportForm.test.tsx

```diff
@@ -206,6 +206,31 @@ describe("The ProjectMilestoneReportForm", () => {
     expect(screen.getAllByText("Remove")).toHaveLength(3);
   });
 
+  it("renders the tooltips for the mock form", () => {
+    componentTestingHelper.loadQuery();
+    componentTestingHelper.renderComponent();
+
+    expect(
+      screen.getAllByLabelText(/maximum-amount-this-milestone-tooltip/)
+    ).toHaveLength(2);
+
+    expect(
+      screen.getAllByLabelText(/total-eligible-expenses-tooltip/)
+    ).toHaveLength(1);
+
+    expect(
+      screen.getAllByLabelText(/gross-payment-amount-this-milestone-tooltip/)
+    ).toHaveLength(2);
+
+    expect(
+      screen.getAllByLabelText(/net-payment-amount-this-milestone-tooltip/)
+    ).toHaveLength(2);
+
+    expect(
+      screen.getAllByLabelText(/holdback-amount-this-milestone-tooltip/)
+    ).toHaveLength(2);
+  });
+
```

Comment:
I'm curious why not using `screen.getByRole("tooltip")` here same as other tests.

Reply:
Mean to revisit this before PR but forgot - For some reason ProjectMilestoneReportForm.test.tsx doesn't render the tooltips as a component with the role "tooltip"... I tried playing around with mock data but no luck. Will work on a fix so we can have consistent tests

Reply:
Fixed!